packages/ocamlmig/ocamlmig.5.2-20250202/opam

```diff
@@ -0,0 +1,69 @@
+opam-version: "2.0"
+synopsis: "OCaml source code rewriting tool"
+description:
+  "Ocamlmig is a command line tool to rewrite ocaml source code, especially to make updating to newer interfaces easier."
+maintainer: ["Valentin Gatien-Baron <valentin.gatienbaron@gmail.com>"]
+authors: ["Valentin Gatien-Baron <valentin.gatienbaron@gmail.com>"]
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+homepage: "https://github.com/v-gb/ocamlmig"
+doc: "https://github.com/v-gb/ocamlmig/blob/main/README.md"
+bug-reports: "https://github.com/v-gb/ocamlmig/issues"
+depends: [
+  "ocaml" {>= "5.2" & < "5.3"}
+  "dune" {>= "3.15"}
+  "base"
+  "core"
+  "core_unix"
+  "csexp"
+  "ppx_partial"
+  "ocaml" {>= "4.08"}
+  "alcotest" {"1" = "0" & >= "1.3.0"}
```

Comment:
```suggestion
  "alcotest" {"1" = "0" & >= "1.3.0"}
```
Why you have these funny bounds? If you want you could use `{with-dev-setup}` and make them available only when invoking opam with that flag

Reply:
The package vendors (a modified copy of) ocamlformat, which means the dependencies section of my opam file copies the dependencies section from ocamlformat's opam file, which is where the alcootest dependency and others come from.
In the ocamlformat package, alcootest is listed as `{with-dev-setup & >= "1.3.0"}`, but that doesn't make sense for my package, since my tests don't use alcootest. So I mechanically replace all the `with-dev-setup` that come from ocamlformat with `false`. But since syntactically that's not allowed (not sure if that's a shortcoming of dune-project or opam itself), instead of `false`, I use `"1" = "0"` which means the same thing :/.

Reply:
The import of ocamlformat's dependencies is done by a script. Dropping the dependencies entirely would involve parsing opam dependencies, which is much more involved than a call to sed.

Reply:
Actually it would involve parsing ocamlformat's dune-project, but the general point is the same.

Reply:
I meant more if it was ok for me to remove them here :)